windows/runner.ps1

```diff
@@ -84,9 +89,19 @@ param (
 
     [Parameter()]
     [switch]
-    $CpuUtilizationOnlyValue
+    $CpuUtilizationOnlyValue,
+
+    [Parameter()]
+    [switch]
+    $Measurements
 )
 
+# If the -Measurements flag is used, set -CPUUtilizationOnlyValue and -CPUUtilizationTimeout 1
+if ($Measurements) {
```

Comment:
Nice! This will avoid any potential confusion and also it's easier to use now. Curious what happens if someone runs MS4W like `./runner.ps1 -UserName Administrator -Measurements -CpuUtilizationTimeout 30`? Does the CPU util timeout you set here (`1`) get the priority or user's explicitly set timeout (`30`) do?

Reply:
Answered in slack but cross posting for visibility.

When running with -Measurements the CpuUtilizationTimeout is always set to 1, regardless of what the user passes in as a flag. In future, I recommend we remove -CpuUtilizationTimeout and -CpuUtilizationOnlyValue entirely from both version of MS. The distinction should be between gather info on servers, and gathering point in time cpu data, and we can use -Measurements to make that distinction.